transcribe/utils.py

```diff
@@ -7,6 +7,20 @@
 
 import jiwer
 
+base_csv_columns = [
+    "file",
+    "language",
+    "runtime",
+    "wer",
+    "mer",
+    "wil",
+    "wip",
+    "hits",
+    "substitutions",
+    "insertions",
+    "deletions",
+]
+
```

Comment:
This defines the order of columns in the output, so that the order of keys in the row dictionaries doesn't determine it.